index.js

```diff
@@ -22,10 +22,11 @@ function fixOrientation (url, opts, fn) {
   
   try { tags = EXIF.readFromBinaryFile(buf.buffer) } catch (err) {}
 
-  var toRotate = tags.Orientation
+  var supportedOrientations = [3,6,8];
+
+  var toRotate = tags.Orientation;
```

Comment:
does this code run? Should have the semicolon at the end

Reply:
Not sure I understand. The line without the semi colon is from the initial commit. It has been removed and replaced with the one with the semi colon.

Reply:
yes but the line after starts with `&&`

Reply:
<img width="469" alt="screenshot 2019-02-11 at 20 01 01" src="https://user-images.githubusercontent.com/10247/52586419-c41fad00-2e37-11e9-9fee-7877bd7967a2.png">

Reply:
Oops, broke it fixing the indentation. 

Just pushed the fix.